packages/rpc-core/src/rpc-methods/common.ts

```diff
@@ -123,3 +128,27 @@ export type TokenAccount = Readonly<{
     closeAuthority?: Base58EncodedAddress;
     extensions?: unknown[];
 }>;
+
+type RewardBase = Readonly<{
+    /** The public key of the account that received the reward */
+    pubkey: Base58EncodedAddress;
+    /** number of reward lamports credited or debited by the account */
+    lamports: LamportsPlusOrMinusAsI64Unsafe;
+    /** account balance in lamports after the reward was applied */
+    postBalance: LamportsUnsafeBeyond2Pow53Minus1;
+}>;
+
+export type Reward =
+    | (RewardBase &
+          Readonly<{
+              /** type of reward */
+              rewardType: 'fee' | 'rent';
+          }>)
+    /** Commission is present only for voting and staking rewards */
+    | (RewardBase &
+          Readonly<{
+              /** type of reward */
+              rewardType: 'voting' | 'staking';
+              /** vote account commission when the reward was credited */
+              commission: number;
```

Comment:
Looks like we need a bigint exception for this too
Maybe `['rewards', KEYPATH_WILDCARD, 'commission']`?